google/cloud/bigtable/client.py

```diff
@@ -161,7 +161,7 @@ def __init__(
     ):
         if client_info is None:
             client_info = client_info_lib.ClientInfo(
-                client_library_version=bigtable.__version__,
+                client_library_version=f"{bigtable.__version__}-legacy",
```

Comment:
Thinking about this a bit more, Lets leave the old agent the same in case there are dashboard that care about it. Instead have the new clients be suffixed with sync & async

Reply:
Sounds good, reverted